cg/store/api/find_business_data.py

```diff
@@ -248,13 +248,22 @@ def get_sequenced_samples(self, family_id: str) -> List[Sample]:
         samples: List[Sample] = self.get_samples_by_case_id(family_id)
         return [sample for sample in samples if sample.sequencing_qc]
 
-    def find_family(self, customer: Customer, name: str) -> Family:
-        """Find a family by family name within a customer."""
-        return self.Family.query.filter_by(customer=customer, name=name).first()
+    def get_case_by_name_and_customer(self, customer: Customer, name: str) -> Family:
```

Comment:
```suggestion
    def get_case_by_name_and_customer(self, customer: Customer, case_name: str) -> Family:
```
To distinguish from customer name and make it very explicit what we expect as input